### PR TITLE
Neue hot module updates

### PR DESCRIPTION
**Pull request by sbsmith86**
This PR makes some updates to neue that were introduced with the hot module work happening in DoSomething app. 

* Added a `-center` option to the `figure -left` pattern so that we can have text vertically centered next to the media.

* Added a new `.avatar` pattern to show a rounded image. 

@DoSomething/front-end

### CODE REVIEW

File: scss/neue.scss [hunk 1]
```diff
@@ -49,6 +49,7 @@
 @import "_components/spinner";
 @import "_components/video";
 @import "_components/waypoints";
+@import "_components/avatar";
```
Comment (DFurnes) [inline scss/neue.scss]: This is a kinda silly nitpick, but can you move this to the top of the list here & in the pattern library file so it stays alphabetical? :grimacing:

File: scss/_components/_avatar.scss [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+//
+// Markup:
+//    <div class="avatar">
+//      <img src="http://placekitten.com/g/300/300" />
+//    </div>
+//
+// Styleguide Avatar
+
+.avatar {
+  border-color: white;
```
Comment (DFurnes) [inline scss/_components/_avatar.scss]: Is this line necessary?

**Comment by DFurnes** [comment]
Looking good! Two tiny Qs but otherwise 100% :ship:-able.

File: scss/_components/_avatar.scss (hunk 2 above)
Comment (sbsmith86) [inline scss/_components/_avatar.scss]: nopppe lol my b

**Pull request merged**